stories/Components/UIcomponents/Gallery/ParallaxGallery/ParallaxGallery.jsx

```diff
@@ -16,9 +16,11 @@ export const ParallaxGallery = ({
   const backgrounds = ["Gray"];
   background = backgrounds.includes(Background) ? Background.toLowerCase() : '';
 
+  const cls = (...classes) => (classes.filter(Boolean).length > 0) ? classes.filter(Boolean).join(' ') : null;
+
   return (
     <div className="grid-x grid-margin-x parallax-gallery-row">
-      <div className={['medium-6 cell parallax-gallery-row__left-col', `${[`${background}`]}`].join(' ').trim()}>
+      <div className={cls('medium-6', 'cell', 'parallax-gallery-row__left-col', `${[`${background}`]}`)}>
```

Comment:
@Shamimkhan321 Remove `-row` and `__left-col` prefix everywhere in `parallax-gallery` and just use `parallax-gallery`. 
Eg. 
`.parallax-gallery`, `.parallax-gallery-left`, `.parallax-gallery-right`, `.parallax-gallery-content` and so on.